keyboards/handwired/polykybd/lang/_gen_alpha_cols.py

```
@@ -0,0 +1,157 @@
+
+Outputs: /tmp/alpha_cols.json, /tmp/alpha_named.json, /tmp/alpha_fonts.json
+"""
+import json, re
+from fontTools.ttLib import TTFont
+F="/home/user/qmk_firmware/keyboards/handwired/polykybd/fonts/"
+def cmap(p): return set(TTFont(F+p).getBestCmap().keys())
+CM={"ka":cmap("noto-sans-georgian/NotoSansGeorgian-Regular.ttf"),
+    "hy":cmap("noto-sans-armenian/NotoSansArmenian-Regular.ttf"),
+    "zh":cmap("noto-sans-tc/NotoSansTC-Regular.ttf")}
+
+ROW={}; L="ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)